airbyte-integrations/connectors/source-shopify/Dockerfile

```diff
@@ -28,5 +28,5 @@ COPY source_shopify ./source_shopify
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.22
+LABEL io.airbyte.version=0.1.23
```

Comment:
Please bump the version up to `0.1.24`, to avoid conflicts.

Reply:
done